From eea304cfac3c0fac0afc5a7c84e59b4de2aa683d Mon Sep 17 00:00:00 2001
From: Christophe Dumez <christophe.dumez@intel.com>
Date: Fri, 3 Jun 2011 09:11:28 +0300
Subject: [PATCH 03/18] QtContacts: Add missing entries to schema definition

QContactOrganization::FieldAssistantName and
QContactOrganization::FieldRole are missing from the schema
definitions although those fields exist since QtMobility 1.0.

One side effect of this is that
QContactManager::compatibleContact() will drop these fields.

Upstream bug: QTMOBILITY-1592.
---
 src/contacts/qcontactmanagerengine.cpp |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/contacts/qcontactmanagerengine.cpp b/src/contacts/qcontactmanagerengine.cpp
index 562148b..912a333 100644
--- a/src/contacts/qcontactmanagerengine.cpp
+++ b/src/contacts/qcontactmanagerengine.cpp
@@ -893,6 +893,8 @@ QMap<QString, QMap<QString, QContactDetailDefinition> > QContactManagerEngine::s
     fields.insert(QContactOrganization::FieldName, f);
     fields.insert(QContactOrganization::FieldLocation, f);
     fields.insert(QContactOrganization::FieldTitle, f);
+    fields.insert(QContactOrganization::FieldAssistantName, f);
+    fields.insert(QContactOrganization::FieldRole, f);
     f.setDataType(QVariant::Url);
     fields.insert(QContactOrganization::FieldLogoUrl, f);
     f.setDataType(QVariant::StringList);
-- 
1.7.10.3

